### PR TITLE
cli: support to serve pipcook artifact

### PR DESCRIPTION
**Pull request by FeelyChau**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2,6 +2,42 @@
 
 The Pipcook Project is an open-source toolkit to enable and accelerate the intelligentization of front-end engineering for Web developers.
 
+# Usage
+Using Pipbook for machine learning development is very simple. It only takes four steps: install, train, test and deploy.
```
Comment (yorkie) [inline docs/README.md:6]:
```suggestion
Using Pipcook for machine learning development is very simple. It only takes four steps: install, train, test, and deploy.
```

File: docs/README.md [hunk 2]
```diff
@@ -2,6 +2,42 @@
...
+
+Install the [Pipcook][] command-line tool:
+
+```shell
+$ npm install -g @pipcook/cli
+```
+
+Then train from a [pipeline](./example/pipelines/):
```
Comment (yorkie) [inline docs/README.md:14]:
```suggestion
Then train from anyone of those [pipelines](./example/pipelines/):
```

File: docs/README.md [hunk 3]
```diff
@@ -2,6 +2,42 @@
...
+```
+
+Predict the category of a image:
+
+```shell
+$ pipcook predict ./output/image-classification-mobilenet.json -s ./output/data/validation/blurBackground/71197_223__30.7_36.jpg
+✔ Origin result:[{"id":1,"category":"blurBackground","score":0.9998120665550232}]
+```
+
+The category is `blurBackground`.
```
Comment (yorkie) [inline docs/README.md:27]: We need to add a tip where does the category from.

File: docs/README.md [hunk 4]
```diff
@@ -2,6 +2,42 @@
...
+```shell
+$ pipcook serve ./output/image-classification-mobilenet.json
+ℹ preparing framework
+ℹ preparing scripts
+ℹ preparing artifact plugins
+ℹ initializing framework packages
+Pipcook has served at: http://localhost:9091
+```
+
+Then you can open the browser and try your image classification service.
```
Comment (yorkie) [inline docs/README.md:39]:
```suggestion
Then you can open the browser and try your image classification server.
```

The server is corresponding to the `serve` :)

File: packages/cli/package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipcook/cli",
-  "version": "2.1.1",
+  "version": "2.1.2-serve-beta.0",
```
Comment (yorkie) [inline packages/cli/package.json:3]: This version is informal, please use 2.1.2 :)

File: packages/costa/package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipcook/costa",
-  "version": "2.1.1",
+  "version": "2.1.2-serve-beta.0",
```
Comment (yorkie) [inline packages/costa/package.json:3]: Ditto.

File: packages/costa/package.json [hunk 7]
```diff
@@ -17,7 +17,7 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@pipcook/core": "^2.1.0",
+    "@pipcook/core": "^2.1.2-serve-beta.0",
```
Comment (yorkie) [inline packages/costa/package.json:20]: Ditto.

**Review by yorkie — CHANGES_REQUESTED** [review]

File: packages/cli/package.json (hunk 5 above)
Comment (FeelyChau) [reply packages/cli/package.json:3]: It should be 2.1.1 unless we want to bump it.

File: README.md [hunk 8]
```diff
@@ -96,18 +96,32 @@ Install the command-line tool for managing [Pipcook][] projects:
...
+This dataset specfied by the pipeline includes 2 categories image: avatar and blurBackground.
+After training, we can predict the category of a image:
 
 ```shell
 $ pipcook predict ./output/image-classification-mobilenet.json -s ./output/data/validation/blurBackground/71197_223__30.7_36.jpg
 ✔ Origin result:[{"id":1,"category":"blurBackground","score":0.9998120665550232}]
 ```
-The category is `blurBackground`.
+
+The input is a `blurBackground` image from the vlidation dataset. And the model determines that its category is `blurBackground`.
```
Comment (yorkie) [inline README.md:112]:
```suggestion
The input is a `blurBackground` image from the validation dataset. And the model determines that its category is `blurBackground`.
```

Comment (FeelyChau) [reply README.md]: Fixed.

**Review by yorkie — APPROVED** [review]

**Pull request merged**